xbmc/epg/EpgContainer.h

```diff
@@ -312,6 +313,7 @@ namespace EPG
     //@{
     bool         m_bIsUpdating;            /*!< true while an update is running */
     bool         m_bIsInitialising;        /*!< true while the epg manager hasn't loaded all tables */
+    bool         m_bIsRestarting;          /*!< true while the pvr manager is restarting (e.g when resuming after system suspend) */
```

Comment:
the ones above have it in their names too, but `Is` is superfluous, that's why it's a bool -> `m_bRestarting`